main.py

```diff
@@ -0,0 +1,138 @@
+            aliased.writerow(header)
+            for card in cards:
+                card_number = card[1]
+                print(f'Aliasing card number {card_number[6:]}******{card_number[:4]}')
+                r = requests.post(f'https://{vault_id}.{environment}.verygoodproxy.com/post',
+                                  json={'account_number': card_number}
+                                  )
+                alias = json.loads(r.json()['data'])['account_number']
+                aliased.writerow([card[0], alias, card[2], card[3]])
+    print(f'Successfully aliased raw data in vault {vault_id}. See "csv/aliased_cards.csv" for created aliases.')
+
+
+def enroll_data(aliased=False):
+    file_name = "csv/aliased_cards.csv" if aliased else "csv/card_numbers.csv"
+
+    print(f'Starting one-time enrollment process from file "{file_name}".')
+    with open(f'{file_name}') as cards_file:
+        with open('csv/updated_cards.csv', 'w') as updated_cards_file:
+            cards = csv.reader(cards_file)
+            next(cards)
+            header = ['Id', 'Name', 'CardNumber', 'ExpirationMonth', 'ExpirationYear']
+            updated = csv.writer(updated_cards_file)
+            updated.writerow(header)
+            for card in cards:
+                card_number = card[1]
+                print(f'Enrolling card number {card_number[6:]}******{card_number[:4]}')
+                proxies = {
+                    'https': f'https://{vault_username}:{vault_password}@{vault_id}.{environment}.verygoodproxy.com:8080'}
+                r = requests.post(f'https://calm.{environment}.verygoodsecurity.app/cards',
+                                  json=request_data(card[0], card_number, card[2], card[3]),
+                                  headers={
+                                      'Content-Type': 'application/json',
+                                      'Authorization': f'Bearer {fetch_jwt()}'
+                                  },
+                                  proxies=proxies if aliased else None,
+                                  verify=Path(__file__).resolve().parent / f'certs/{environment}_cert.pem' if aliased else None
+                                  )
+                data = r.json()['data']
+                card_id = data['id']
+                name = data['name']
+                updated_number = data['number']
+                updated_exp_month = data['exp_month']
+                updated_exp_year = data['exp_year']
+                updated.writerow([card_id, name, updated_number, updated_exp_month, updated_exp_year])
+    print(
+        f'Successfully finish one-time enrollment process. '
+        f'See "csv/updated_cards.csv" for enrolled card with their corresponding ID\'s.')
+
+
+if __name__ == "__main__":
+    vault_id = os.getenv('VAULT_ID')
+    vault_username = os.getenv('VAULT_USERNAME')
+    vault_password = os.getenv('VAULT_PASSWORD')
+    environment = os.getenv('ENVIRONMENT') if os.getenv('ENVIRONMENT') else 'sandbox'
```

Comment:
```suggestion
    environment = os.getenv('ENVIRONMENT') or 'sandbox'
```

Reply:
@odovzhenko Could you please apply suggestion in a follow up PR? thanks.

Reply:
Done.